input/resources/au-core-responder.xml

```diff
@@ -986,6 +988,8 @@
     <div>
       <p>The requester <strong>SHALL</strong> provide at least a code value and <strong>MAY</strong> provide both the system and code values.</p>
       <p>The responder <strong>SHALL</strong> support both.</p>
+      <p>The requester <strong>SHOULD</strong> support <code>multipleOr</code>.</p>
```

Comment:
This should be SHALL for both requester & responder.

Reply:
fixed

Reply:
fixed